include/mixer.h

```diff
@@ -242,25 +267,41 @@ class Mixer : public ParamListenerInterface
     &octocopter_x_mixing,
     &Y6_mixing,
     &X8_mixing,
-    &tricopter_mixing,
     &fixedwing_mixing,
-    &passthrough_mixing,
-    &fixedwing_vtail_mixing,
-    &quadplane_mixing,
-    &custom_mixing
+    &fixedwing_inverted_vtail_mixing,
+    &custom_mixing,
   };
 
   // clang-format on
 
+  // Define parameters for the mixer
+  float R_;                      // Motor resistance
+  float rho_;                    // Air density
+  float K_V_;                    // Motor back-emf constant
+  float K_Q_ = 0.01706;          // Motor torque constant
```

Comment:
Was this meant to be a hard-coded constant?

Reply:
Nope. I don't know why that one escaped. I fixed that.